apps/dashboard/features/dao-overview/hooks/useCountdown.ts

```diff
@@ -89,15 +83,15 @@ function calculateTimeLeft(targetTimestamp?: number): CountdownTime {
     };
   }
 
-  const days = Math.floor(diffMs / MILLISECONDS_PER_DAY);
+  const days = Math.floor(diffMs / 1000);
   const hours = Math.floor(
-    (diffMs % MILLISECONDS_PER_DAY) / MILLISECONDS_PER_HOUR,
+    (diffMs % 1000) / 1000,
   );
   const minutes = Math.floor(
-    (diffMs % MILLISECONDS_PER_HOUR) / MILLISECONDS_PER_MINUTE,
+    (diffMs % 1000) / 1000,
   );
   const seconds = Math.floor(
-    (diffMs % MILLISECONDS_PER_MINUTE) / MILLISECONDS_PER_SECOND,
+    (diffMs % 1000) / 1000,
```

Comment:
Is this correct? This way, hours == minutes == seconds arent they?

Reply:
I agree, MILLISECONDS_PER_MINUTE should be replaced by 60000, not 1000, but I also liked the constants, I don't see any reason to remove them